manifests/base/kustomization.yaml

```diff
@@ -2,14 +2,17 @@ apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 namespace: kubeflow
 resources:
-- crd.yaml
-- cluster-role-binding.yaml
-- cluster-role.yaml
-- deployment.yaml
-- service-account.yaml
-- service.yaml
+  - crd_tfjobs.yaml
+  - crd_mxjobs.yaml
+  - crd_pytorchjobs.yaml
+  - crd_xgboostjobs.yaml
+  - cluster-role-binding.yaml
+  - cluster-role.yaml
+  - deployment.yaml
+  - service-account.yaml
+  - service.yaml
 commonLabels:
-  app: tf-job-operator
-  kustomize.component: tf-job-operator
-  app.kubernetes.io/component: tfjob
-  app.kubernetes.io/name: tf-job-operator
+  app: training-operator
+  kustomize.component: training-operator
+  app.kubernetes.io/component: kubeflow-training
+  app.kubernetes.io/name: training-operator
```

Comment:
This commonLabels can be removed.
We are not using Application CR for Kubeflow more.
Check: https://github.com/kubeflow/manifests/issues/1573#issuecomment-858376223.
cc @kubeflow/wg-training-leads @yanniszark